db/seeds.rb

```diff
@@ -41,19 +41,19 @@
                      brand: 'Ride',
                      image: File.open('app/assets/images/holman_bib.png')
                    },
-                   { name: 'GOOD VIBES ZIP HOODIE',
+                   { name: 'ZIP HOODIE',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.